scr/hw6/ArrayDataException.java

```diff
@@ -0,0 +1,13 @@
+package hw6;
+
+public class ArrayDataException extends Throwable implements MyException {
+    private String message;
+
+    public ArrayDataException(String m) {
```

Comment:
What is `m`? Please use fully readable names. In this case `message`

Reply:
how can i write a message again?